web-components/src/[sandbox]/sandbox.ts

```diff
@@ -122,14 +122,6 @@ export class Sandbox extends LitElement {
       <md-theme class="theme-toggle" id="app-theme" ?darkTheme=${this.darkTheme} ?lumos=${this.lumos}>
         ${this.themeToggle()}
         <elix-list-explorer class="explorer">
-
-          <div class="container" aria-label="md-table-advanced">
-            <h2>md-table-advanced</h2>
-            <sass-stats component="table-advanced">
-              ${tableAdvancedTemplate}
-            </sass-stats>
-          </div>
-
```

Comment:
Here too, seems unrelated to ComboBox, but not clear if it is cleanup or accidental removal

Reply:
Probably just put in in alphabetical order here. I agree that I was confusing so many changes were being made for table-advanced

Reply:
@elenagiere @boiarchukk Aha, yes that's it!